mappings/net/minecraft/recipe/book/RecipeBook.mapping

```diff
@@ -7,11 +7,15 @@ CLASS xq net/minecraft/recipe/book/RecipeBook
 	FIELD f furnaceFilteringCraftable Z
 	METHOD a isGuiOpen ()Z
 	METHOD a isFilteringCraftable (Latx;)Z
+	METHOD a add (Lqc;)V
 	METHOD a copyFrom (Lxq;)V
 	METHOD a setGuiOpen (Z)V
 	METHOD b isFilteringCraftable ()Z
+	METHOD b contains (Layw;)Z
 	METHOD b setFilteringCraftable (Z)V
 	METHOD c isFurnaceGuiOpen ()Z
 	METHOD c setFurnaceGuiOpen (Z)V
 	METHOD d isFurnaceFilteringCraftable ()Z
+	METHOD d shouldDisplay (Layw;)Z
 	METHOD d setFurnaceFilteringCraftable (Z)V
+	METHOD e didDisplay (Layw;)V
```

Comment:
didDisplay? Void?... Is this an event-handler-like? If so, onDisplayed?

Reply:
Yeah I guess it's a bit event-y. RecipeDisplayListener#onRecipesDisplayed and friends.